updater/manager/AddonManager.py

```diff
@@ -9,6 +12,11 @@
 
 from updater.site import SiteHandler
 
+def format_path(path: str):
+    if sys.platform != 'linux' or 'Microsoft' not in platform.uname()[3]:
```

Comment:
I just ran a small test on Mac, to see what the output of these commands looks like:

`platform.uname()[3]` looks equivalent to `platform.version()`, and according to the docs, that's true:

> `platform.version()`
>     Returns the system’s release version, e.g. '# 3 on degas'. An empty string is returned if the value cannot be determined.
> 
> `platform.uname()`
>     Fairly portable uname interface. Returns a tuple of strings (system, node, release, version, machine, processor) identifying the underlying platform.

So I think this can be simplified to read like:
```python
if 'Microsoft' in platform.version() and sys.platform == 'linux':
    return subprocess.check_output(['wslpath', path], text=True)
else:
    return path
```

Reply:
I agree `platform.version()` should be in use here. I already made the changes requested.

Should I keep the `text` parameter of `subprocess.check_output()`? It was added in Python 3.7, but I see you are requiring 3.6+. Could simply change that to `universal_newlines`.

Reply:
That's a good point. I originally bumped the requirement to 3.6 so that I could bring in f-strings. I have a vision of a potential switch to use `async/await` in place of manually threading, which will require 3.7.

Let's go with 3.7.